Source/Generic/ECTestCase.h

```diff
@@ -9,6 +9,18 @@
 
 @class NSBitmapImageRep;
 
+/// The reporting mode enum when comparing test results
+/// Are differences reported by character, line etc?
+typedef NS_ENUM(NSUInteger, ECTestComparisonMode)
+{
+	ECTestComparisonShowChars,
+	ECTestComparisonShowLines,
+	ECTestComparisonShowLinesIgnoreWhitespace,
+	ECTestComparisonDiff,
+	ECTestComparisonDiffNoJSON,
+	ECTestComparisonDefault
+};
```

Comment:
I don't feel qualified to write the docs for these. This enum was moved because it was clear it was in the wrong header file.